.github/workflows/go.yml

```diff
@@ -14,7 +14,55 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ matrix.go-version }}
-    - name: Checkout code
+    - name: Checkout code to gopath
       uses: actions/checkout@v1
-    - name: Build & Test
-      run: make
\ No newline at end of file
+      with:
+        path: go/src/github.com/hyperledger-labs/fabric-chaincode-wasm
+    - name: Debug dirs and env
+      run: |
+        pwd
+        echo "###### ENV #######"
+        env
+        echo "###### GO ENV #######"
+        go env
+        echo ${GOPATH}
+      env:
+        GOPATH: /home/runner/work/fabric-chaincode-wasm/go
+    - name: Install ginkgo
+      run: |
+        go get -u github.com/onsi/ginkgo/ginkgo
+      env:
+        GOPATH: /home/runner/work/fabric-chaincode-wasm/go
+    - name: Get HLF
+      working-directory: ./../..
+      run: |
+        mkdir hyperledger
+        cd hyperledger
+        mkdir fabric
+        cd fabric
+        pwd
+        git clone https://github.com/hyperledger/fabric . --branch v1.4.0 --single-branch --depth 1
+    - name: Install ccenv couchdb kafka zookeeper
+      working-directory: ./../../hyperledger/fabric
+      run: |
+        docker pull hyperledger/fabric-ccenv:amd64-1.4.0
+        docker tag hyperledger/fabric-ccenv:amd64-1.4.0 hyperledger/fabric-ccenv:amd64-latest
+        docker pull hyperledger/fabric-zookeeper:0.4.14
+        docker tag hyperledger/fabric-zookeeper:0.4.14 hyperledger/fabric-zookeeper:latest
+        docker pull hyperledger/fabric-kafka:0.4.14
+        docker tag hyperledger/fabric-kafka:0.4.14 hyperledger/fabric-kafka:latest
+        docker pull hyperledger/fabric-couchdb:0.4.14
+        docker tag hyperledger/fabric-couchdb:0.4.14 hyperledger/fabric-couchdb:latest
+      env:
+        GOPATH: /home/runner/work/fabric-chaincode-wasm/go
+    - name: Build & Unit Test
+      run: |
+        make
+    - name: Integration test
+      working-directory: ./integration/e2e
+      run: |
+        pwd
+        go test
```

Comment:
do you want to use ginkgo here instead since you install it above?

Reply:
go bin is not in path, that's why I avoided it. Is there any advantage of using ginkgo over go test?

Reply:
plus it needs gcc: https://github.com/kleash/fabric-chaincode-wasm/runs/268514684

Reply:
No real advantage, just curious as to why you were installing it above.

Reply:
Oh ok. I think I tried using ginkgo first and later replaced it with go test. But didn't remove the installation part.

I have removed ginkgo installation now. You are right, it's an extraneous step.